content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -165,6 +166,77 @@ public void init() {
 		// Populate content upload headers used in uploadExternalContent
 		CONTENT_UPLOAD_HEADERS.putAll(BASE_HEADERS);
 		CONTENT_UPLOAD_HEADERS.put(HEADER_CONTENT, CONTENT_TYPE_BINARY);
+
+		try {
+			String webhookUrl = getWebhookUrl(Optional.empty());
+			if (!webhookUrl.contains("localhost")) {
+				webhookId = setupWebhook(webhookUrl);
```

Comment:
need to check first if an existing webhook is already set up for that URL

1) You'll want to get the whole list of webhooks (not just the one webhook based on ID)
2) Check each registered webhook and make sure that the correct URL exists
3) If it doesn't exists, then add a new webhook for the correct URL